CHANGELOG.md

```diff
@@ -1,13 +1,11 @@
-## v0.9.4 - ....-..-.. - Dynamic Records Beta
-
-* Dynamic Records (more powerful/flexible replacement for geo)
-   * Will support A, AAAA, and CNAME out the gate and include the ability to
-     weight records. It should provide a foundation for further suppport
-     if/when needed.
-   * Major refactoring and improvements to validation of the Record hierarchy,
-     things are much more consisntely implemented now and error messages should
-     be more actionable/clear. Both the base values and dynamic values use the
-     same validatio logic.
+## v0.9.4 - 2019-01-28 - The one with a bunch of stuff, before the big one
+
+* A bunch of "dynamic" stuff that'll be detailed in the next release when
+  providers actually support it :grin:
+* Route53Provider adds support for using session tokens
+* Added support for proxying Cloudflare ALIAS records
```

Comment:
I think I can set proxying for all Cloudflare records!

Reply:
Speaking of which, there may be a tiny bit of confusion on this flag until https://github.com/github/octodns/pull/292 is shipped

Reply:
> Speaking of which, there may be a tiny bit of confusion on this flag until #292 is shipped

Yeah. I don't think it's going to be any worse than it already was as a result of this :ship: since proxy and similar stuff existed before, but I agree. I'd like to return to #292 or something broader that does better formatting in general as it'd help a lot with that and with the more complex `geo`/`dynamic` records as well.

Reply:
Oh yeah https://github.com/github/octodns/pull/307 has bits and pieces of that, but it turned into a mess that I didn't have the space to sort out at the time. I'll probably pick it back up eventually.